app/scripts/views/confirm.js

```diff
@@ -41,14 +44,23 @@ function (Cocktail, FormView, BaseView, Template, p, AuthErrors, Constants,
     },
 
     context: function () {
-      return {
+      var context = {
         email: this.getAccount().get('email')
       };
+
+      context.verificationExperiment = this._verificationExperiment.getContext();
+
+      if (this._verificationExperiment.is('openGmail')) {
```

Comment:
Did we fix the problem where `context` was called once for each variable in the template? If not, this will cause lines 53-55 to be called multiple times. Will that clutter the logs?